internal/postrs/proof.go

```diff
@@ -152,7 +144,49 @@ func (v *Verifier) Close() error {
 	return nil
 }
 
-func (v *Verifier) VerifyProof(proof *shared.Proof, metadata *shared.ProofMetadata, logger *zap.Logger, cfg Config, scryptParams ScryptParams) error {
+type verifyOptions struct {
+	mode C.VerifyMode
+}
```

Comment:
I don't think an options struct should contain C types. I think would be safer to use go types in go and only convert them right before calling the actual C function.

Reply:
Updated